api/api.go

```diff
@@ -52,6 +76,10 @@ func registerRoutes(ws *gin.Engine, elrondFacade middleware.ElrondHandler) {
 	txRoutes.Use(middleware.WithElrondFacade(elrondFacade))
 	transaction.Routes(txRoutes)
 
+	if elrondFacade.(MainApiHandler).PrometheusMonitoring() {
```

Comment:
We should extract `elrondFacade.(MainApiHandler)` in a variable, something like `apiHandler, ok := elrondFacade.(MainApiHandler)`. Then check if ok is true, otherwise this code will panic if elrondFacade can't be casted to MainApiHandler

Reply:
done